dataland-frontend/tests/e2e/utils/Auth.ts

```diff
@@ -14,7 +14,7 @@ export function logout(): void {
     .wait(1000)
     .url()
     .should("eq", getBaseUrl() + "/")
-    .get("button[name='login_dataland_button']")
+    .get("a[aria-label='Login to preview account']")
```

Comment:
The landing page does not speak of a "preview" product anymore.
Shall we remove the "preview" word from stuff like this?

Reply:
Without a data privacy text and no cookie banner I would leave it as is.